webpack.config.js

```diff
@@ -67,6 +67,7 @@ const blastScriptConfig = {
         { from: path.join(nodeModules, '/datatables.net-colreorder-dt/css/colReorder.dataTables.css'), to: blastStyles},
         { from: path.join(nodeModules, '/datatables.net-bs/css/dataTables.bootstrap.css'), to: blastStyles},
         { from: path.join(nodeModules, '/datatables.net-dt/css/jquery.dataTables.css'), to: blastStyles},
+        { from: path.join(nodeModules, '/jquery-ui-1.11.1/jquery-ui.css'), to: blastStyles},
```

Comment:
You need to add **jquery-ui** into **package.json** and remove the **jquery-ui.css**.

Reply:
![package2](https://user-images.githubusercontent.com/33312760/47362915-8ee22a00-d6a3-11e8-8bcc-04d361e2049f.PNG)
as far as i can see , it's already iclude jquery in jason.package should i add the extra "jquery-ui"?

Reply:
Based on this line, you install **jquery-ui.css ** from another npm package called **jquery-ui-1.11.1**.

Reply:
i find out that there is a jquery-ui.css in juqery-ui-themes but its version is 1.12.0, i will also figure it if i need to install the version 1.11.1 or just use 1.12.0